lib/still/compiler/traverse.ex

```diff
@@ -39,5 +37,17 @@ defmodule Still.Compiler.Traverse do
     |> List.flatten()
   end
 
+  defp compile_files(files, callback) do
+    files
+    |> Enum.map(fn file ->
+      Task.async(fn ->
+        callback.(file)
+      end)
+    end)
+    |> Enum.each(fn task ->
+      Task.await(task, :infinity)
+    end)
```

Comment:
This is equivalent to `Task.await_many/2`

Reply:
I didn't know about that :O

Reply:
It seems that it was only introduced in elixir 1.11, so I dropped our support for 1.10, I guess it doesn't really matter at the moment